docs/lake/design.md

```diff
@@ -509,6 +513,20 @@ zed lake delete -p logs <id> <id> <id>
 > whatever reason, it should simply continue on and issue a warning on
 > the query endpoint "warnings channel".
 
+### Revert
+
+The actions in a commit can be reversed with the `revert` command.  This
+command applies the inverse steps in a new commit to the tip of the indicated
+branch.  Any data loaded in a reverted commit remains in the lake but no longer
+appears in the branch.  The new commit may recursively be reverted by an
+additional revert operation.
+
+For example, this command reverts the commit referenced by commit ID
+`<commit>`.
+```
+zed lake revert -p logs <commit>
+```
+
```

Comment:
At the same time I was educated on Slack about `delete` only applying to objects, I was reminded of the existence of `revert`, at which point I noticed it's absent from the doc. I just copied this text pretty much verbatim from the `zed` help text. Feel free to propose more adjustments to suit the narrative nature of this doc.